cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaBridgeCluster.java

```diff
@@ -486,14 +486,6 @@ public PodDisruptionBudget generatePodDisruptionBudget() {
         return PodDisruptionBudgetUtils.createPodDisruptionBudget(componentName, namespace, labels, ownerReference, templatePodDisruptionBudget);
     }
 
```

Comment:
No need for the double empty lines here which happen after you delete the method.